com.unity.transport/Tests/Editor/DataStreamTests.cs

```diff
@@ -69,7 +69,7 @@ public void ReadIntoExistingByteArray()
                 dataStream.Write((byte) 'c');
                 var reader = new DataStreamReader(dataStream, 0, dataStream.Length);
                 var readerCtx = default(DataStreamReader.Context);
-                reader.ReadBytesIntoArray(ref readerCtx, ref byteArray, dataStream.Length);
+                reader.ReadBytesIntoArray(ref readerCtx, byteArray, dataStream.Length);
```

Comment:
Are you sure that it works?
ByteArray used in _76 line_, but after _62 line_ newer chaneged. 
The _72 line_ once is where the **byteArray** can be modified.

Reply:
Yes, it works. The test wouldn't have passed if it didn't work. You don't need a ref to an array to modify the array. You only need a ref to an array if you want to point to an entirely different array.

That said, it doesn't look like Unity is interested in PRs, so I'm going to drop this.